task2.js

```diff
@@ -0,0 +1,24 @@
+// Create a function that takes a "base number" as an argument.
+// This function should return another function which takes a new argument,
+// and returns the sum of the "base number" and the new argument.
+// plusFive(2); // ➞ 7
+// plusTen(188); // ➞ 198
+// plusFive(plusTen(0)); // ➞ 15
+
+const { isValidNumber } = require("./utils");
+
+function plusFactory(num) {
+  return (num2) => {
+    if (!isValidNumber(num) || !isValidNumber(num2)) {
```

Comment:
divide validation. Divide it for two `if`. One inside `plusFactory` second should be inside `children `function